readme.md

```diff
@@ -1 +1,78 @@
-[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
+# Deploying Azure Resources 
+This project provides you with the scripts needed to deploy the necessary resources for a Microsoft Speech Studio demo. 
+
+## Things you need first 
+1. An [Azure](https://azure.microsoft.com/) subscription. 
+2. PowerShell
+
+## Deploying Azure Resources
+1. Click the botton blow:<br/>[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)<br> 
+
+2. A new window should pop up that prompts you to log into your Azure account. 
+
+3. Once logged in your should be greeted with the below screen:
+![Landing Page](doc/LandingPage.png)
+
+4. Make sure the correct "Directory" and "Subscription" are selected.
+
+5. Next, you are able to edit the name of "Resource Group Name", "Site Name", and input a value for "Signalr_service_name". You are also able to choose the "SiteLocation" of where you want these resources to be deployed. 
+<br/>\*Recommendation: Make the names easy to remember. You will be using them later.<br>
+
+6. Click "Next" at the bottom of the page.
+
+7. The next page will list the Azure resources that will be deployed. Click "Deploy" to continue. 
+![List of Resources](doc/ResourceList.png)
+
+8. It will take a few minutes for all the resources to be deployed, but once it is complete you should see the following page. 
+![Completed Deployment](doc/ResourceFinish.png)
+
+9. You are now ready to deploy the Speech Services. 
+
+## Deploying Speech Service
+1. Clone the [VoiceTest](https://github.com/negativeeddy/voicetest) repo to your local machine.
+
+2. In PowerShell, Navigate to the "deployment" folder of the cloned repo and run the createSpeechApp.ps1 script.
+ ````
+     \voicetest-master\voicetest-master> cd deployment
+ ````
+ ````
+     .\createSpeechApp.ps1
+ ````
+3. The script will prompt you for the following information(all will be found in your Azure Portal):
+
+   •	siteName – whatever you used for the sitename in "Deploying Azure Resources"
+   
+   •	speechResourceKey – the key from the <siteName>-speech resource
+   
+   •	luisauthoringkey – the key from the <siteName>-luisauthoringkey resource
+   
+   •	your subscription Id
+
+4. Once deployed your should get the following message:
+````
+Speech commands have been published.
+````
+
+
+## Testing
```

Comment:
I like this section. Later when we add the UWP, this needs to stay at the end for a "more on this stuff" kind of thing

Reply:
Sounds good